shell/test/specs/demo-tests.js

```diff
@@ -181,7 +181,7 @@ function waitForStillness() {
     }
     return matches > desiredMatches;
   };
-  browser.waitUntil(noGlow, 20000, `the glow can't stop won't stop`, 500);
```

Comment:
Do we want to check in this boost here (and below?)

Reply:
No, I'm trying to debug the wdio tests; I can't launch them locally, and on Michael's machine they run to completion. Timing out was one hypothesis.